### ISSUE TITLE
Extremely slow and frequently freezes

### ISSUE THREAD

**Issue by adigold** [stackblitz/bolt.new#420]
### Describe the bug

Everything is sooooooooo slow, and the pages keep freezing, even with a pro account. What a bummer! I've canceled my subscription.

### Link to the Bolt URL that caused the error

-

### Steps to reproduce

Everything is sooooooooo slow, and the pages keep freezing, even with a pro account. What a bummer! I've canceled my subscription.

### Expected behavior

-Everything is sooooooooo slow, and the pages keep freezing, even with a pro account. What a bummer! I've canceled my subscription.

### Screen Recording / Screenshot

-

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [e.g. 91.1]

### Additional context

-

**Comment by tpistalu** [comment]
same here, unless we do something wrong?

**Comment by manfrin** [comment]
Love bolt, hate using stackblitz. Out of curiosity I switched to chrome and it works fine. They are apparently not targeting firefox which is extremely disappointing.

**Comment by achintya175** [comment]
yes, I have also tried bolt but it taking hours to do simple work

**Comment by urbansoulhunter** [comment]
same here.
